digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1934,4 +1932,25 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
       )
     }
   }
+
+  get("/privateRoads/:municipalityCode") {
+    val municipalityCode = params("municipalityCode").toInt
+    val municipalityName = roadLinkService.municipalityService.getMunicipalityNameByCode(municipalityCode)
+    val cachedRoadLinks = roadLinkService.getTinyRoadLinkFromVVH(municipalityCode).map(_.linkId).toSet
+    val results = roadLinkService.getPrivateRoadsInfoByLinkIds(cachedRoadLinks)
+    val groupedResults = roadLinkService.groupPrivateRoadInformation(results)
```

Comment:
should be move to roadlinkService

`val cachedRoadLinks = roadLinkService.getTinyRoadLinkFromVVH(municipalityCode).map(_.linkId).toSet
    val results = roadLinkService.getPrivateRoadsInfoByLinkIds(cachedRoadLinks)
    val groupedResults = = roadLinkService.groupPrivateRoadInformation(results)`